### PR TITLE
Replace bloated payments svg with small payments png

### PR DESCRIPTION
**Pull request by jlieb10**
## What does this change?
Replace bloated payments svg with small payments png. Fixes [this PR](https://github.com/guardian/frontend/pull/21289/files)

## What is the value of this and can you measure success?
~400kb -> ~3kb

### Tested

- [X] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js [hunk 1]
```diff
@@ -61,7 +61,13 @@ export const acquisitionsBannerFivTemplate = (
                     ${mobileButton}${arrowWhiteRight.markup}
                 </a>
                 <div class="engagement-banner__payment-logos engagement-banner__payment-logos--fiv">
-                    ${paymentMethods.markup}
+                    <img
+                        src="${config.get(
+                            'images.acquisitions.paypal-and-credit-card',
+                            ''
+                        )}"
+                        alt="payment options"
```
Comment (SiAdcock) [inline static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js]: Small accessibility request: can we change this to "Accepted payment methods: Visa, Mastercard, American Express and Paypal". Or something similar?

Comment (jlieb10) [reply static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js]: 👍

**Comment by paperboyo** [comment]
Just because it looks like svg was rasterised at 100% so might look blurry on “retina” screens, here it is twice the dimensions (422px wide) . And smaller still.

payment-methods_opt

**Review by paperboyo — APPROVED** [review]
👍

**Comment by johnduffell** [comment]
2.39kb much better, good work all!
👍 🚢

**Pull request merged**